Alignment/OfflineValidation/test/PVValidation_TEMPL_cfg.py

```diff
@@ -1,154 +1,153 @@
-
-     ## SiStrip Lorentz Angle corrections
-     process.conditionsInSiStripLorentzAngleRcd = CalibTracker.Configuration.Common.PoolDBESSource_cfi.poolDBESSource.clone(
-          connect = cms.string('sqlite_file:/afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/MP/MPproduction/mp1025/SiStripLorentzAngleDeco_GR10_v1_offline_BPCorrected.db'),
-          toGet = cms.VPSet(cms.PSet(record = cms.string('SiStripLorentzAngleRcd'),
-                                     tag = cms.string('SiStripLorentzAngleDeco_GR10_v1_offline_BPCorrected'),
-                                     label = cms.untracked.string('deconvolution')
-                                     )
-                            )
-          )
-     
-     process.prefer_conditionsInSiStripLorentzAngleRcd = cms.ESPrefer("PoolDBESSource", "conditionsInSiStripLorentzAngleRcd")
-     
-     ## SiStrip backplane corrections
-     process.conditionsInSiStripConfObjectRcd = CalibTracker.Configuration.Common.PoolDBESSource_cfi.poolDBESSource.clone(
-          connect = cms.string('sqlite_file:/afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/MP/MPproduction/mp1025/SiStripShiftAndCrosstalk_GR10_v2_offline_BPCorrected.db'),
-          toGet = cms.VPSet(cms.PSet(record = cms.string('SiStripConfObjectRcd'),
-                                     tag = cms.string('SiStripShiftAndCrosstalk_GR10_v2_offline_BPCorrected')
-                                     )
-                            )
-          )
-     process.prefer_conditionsInSiStripConfObjectRcd = cms.ESPrefer("PoolDBESSource", "conditionsInSiStripConfObjectRcd")
-     
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, 'GLOBALTAGTEMPLATE', '')
+
+if allFromGT:
+     print ">>>>>>>>>> testPVValidation_cfg.py: msg%-i: All is taken from GT"
 else:
-     print "msg%-i: Primary Vertex Validation: Not applying extra calibration constants!"
+     ####################################################################
+     # Get Alignment constants
+     ####################################################################
+     from CondCore.DBCommon.CondDBSetup_cfi import *
+     process.trackerAlignment = cms.ESSource("PoolDBESSource",CondDBSetup,
+                                             connect = cms.string('ALIGNOBJTEMPLATE'),
+                                             timetype = cms.string("runnumber"),
+                                             toGet = cms.VPSet(cms.PSet(record = cms.string('TrackerAlignmentRcd'),
+                                                                        tag = cms.string('GEOMTAGTEMPLATE')
+                                                                        )
+                                                               )
+                                             )
+     process.es_prefer_trackerAlignment = cms.ESPrefer("PoolDBESSource", "trackerAlignment")
```

Comment:
@ghellwig indeed it's not (actually it's worse: it's deprecated). 
I have in the pipeline another PR to fix it (not only here).